docs/api/0.7.3/client/secrets.md

```diff
@@ -18,7 +21,7 @@ To set a Secret in Prefect Cloud, you can use `prefect.Client.set_secret`, or se
 
 ```graphql
 mutation {
-  setSecret(input: { name: "KEY", value: "VALUE" }) {
+  set_secret(input: { name: "KEY", value: "VALUE" }) {
```

Comment:
I don't feel _too strongly_ about it but I don't think we should update older versioned API documentation

Reply:
Ah, good catch-- got caught up in finding + replacing, didn't notice this was an older file. I'll go ahead and revert.

Reply:
There we go-- should be good to go now.